pom.xml

```diff
@@ -3,7 +3,7 @@
 
   <groupId>com.hcl</groupId>
   <artifactId>appscan-cloud-cli</artifactId>
-  <version>1.0.1-SNAPSHOT</version>
+  <version>1.0.0-SNAPSHOT</version>
```

Comment:
Since version 1.0.0 has already been released, we should keep this as 1.0.1-SNAPSHOT. The deployment process will remove the -SNAPSHOT suffix and update this to 1.0.2-SNAPSHOT when we do the next release.

Reply:
Sure. Fixed this...